### DISCUSSION TITLE
Checkboxes in editor mode

### DISCUSSION THREAD

**Discussion by nezlok** [radekkozak/discordian#19] [Ideas]
Hm, any particular reason why checkboxes are rendered as if they're in preview mode?  It's ... odd, to try and maneuver around them when editing.  Perhaps it's a style choice?

**Comment by radekkozak** [comment]
Hi mate, sorry for delay, i was out of the grid this weekend. Yes, you're right this was more or less a design choice - to bring the Editor and Preview closer wherever possible (next update will make it more so in terms of width, spacing and so on)

I could make it optionable probably in the next release but more people actually enjoy this than have problem with it - **i would love it if you could tell me how this feature creates a problem for you when editing list** - maybe there is a bug indeed /cc @nezlok 

It should be straight forward. I'm testing it on macOS and it's all good there. One prerequisite thought is to have enabled `Smart indent` as i described it in [Beautiful task lists - also in Editor Mode](https://github.com/radekkozak/discordian#task-lists) 

I'm attaching short vid for you: editing lists should be as easy as `Tab` & `Shift-Tab` maneuver


https://user-images.githubusercontent.com/2744747/105686594-b9149900-5ef7-11eb-85fd-6c264f10e142.mov
